inhibitors.py

```diff
@@ -12,103 +12,106 @@
 
 
 def AutoSelect():
-	import platform
```

Comment:
this should be unnecessary, right?

Reply:
Probably - I don't write too much python lately, but I remember scoping imports inside functions to be cut-pasted easily.  I don't have a strong preference here, so take your pick :grin: